eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/command/TeleportToRandomPlayerCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.eternalcode.core.feature.teleport.command;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.configuration.implementation.PluginConfiguration;
+import com.eternalcode.core.notice.NoticeService;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+@Route(name = "teleport-to-random-player", aliases = { "tprp" })
+@Permission("eternalcore.tprp")
+public class TeleportToRandomPlayerCommand {
+
+    private static final Random RANDOM = new Random();
+
+    private final Server server;
+    private final PluginConfiguration pluginConfiguration;
+    private final NoticeService noticeService;
+
+    public TeleportToRandomPlayerCommand(Server server, PluginConfiguration pluginConfiguration, NoticeService noticeService) {
+        this.server = server;
+        this.pluginConfiguration = pluginConfiguration;
+        this.noticeService = noticeService;
+    }
+
+    @Execute
+    @DescriptionDocs(description = "Teleport to random player on server")
+    void execute(Player player) {
+        List<Player> applicablePlayers = this.server.getOnlinePlayers().stream()
+            .filter(players -> this.pluginConfiguration.teleport.includeOpPlayersInRandomTeleport || !players.isOp())
+            .collect(Collectors.toList());
+
+        if (applicablePlayers.isEmpty()) {
+            this.noticeService.create()
+                .player(player.getUniqueId())
+                .notice(translation -> translation.teleport().noPlayerToRandomTeleportFound())
+                .send();
+            return;
+        }
+
+        Player target = applicablePlayers.get(RANDOM.nextInt(applicablePlayers.size()));
+
+        player.teleport(target);
+        this.noticeService.create()
```

Comment:
```suggestion
        player.teleport(target);
        
        this.noticeService.create()
```

Reply:
The player method is strongly associated with sending messages, I try to divide the enters in the code so that they are divided into blocks.

example:
![Przechwytywanie](https://github.com/EternalCodeTeam/EternalCore/assets/65517973/d3382b67-9b22-4777-9b9d-99a952215e92)

Reply:
You can divide the final stage into 2 stages:
- Player teleportation
- Sending a message
Just as I proposed.
In my opinion, it is more readable, but if you want, you can leave it as it is now.